lms/static/styles/components/_FilePickerApp.scss

```diff
@@ -29,9 +27,24 @@ $-space: 1rem; // TODO: replace this with a real variable
   top: calc(50% - #{var.$spinner-size / 2});
 }
 
-.FilePickerApp__document-source-buttons {
-  @include layout.vertical-rhythm($-space * 0.5);
-  display: inline-flex;
-  flex-direction: column;
-  margin-top: 30px;
+.FilePickerApp__actions {
+  // A flex container (row) with two children: a column of buttons and a
+  // "spacer" to take up the rest of the row space not needed by the buttons.
+  // The spacer prevents the button container from stretching full width.
+  display: flex;
+
+  &-buttons {
+    // Display buttons in a column with a vertical margin between each button
+    display: flex;
+    flex-direction: column;
+    & > * + * {
+      // Margin reset on button makes `!important` necessary here
+      margin-top: 0.5em !important;
+    }
+  }
+
+  &-spacer {
```

Comment:
Since this utility isn't doing anything besides `u-stretch` using the utility class would save a little code here. If a bunch of other rules were added to the selector then I think that would better justify a new class.

Reply:
What I was trying to get at in my comment is that I acknowledge that the `.u-stretch` class would have accomplished the same thing, but felt that ergonomically for the time being it was easier to understand if all of the rules were in one place in this particular case. It's easier to look at as a whole and squint and think: "Is this a pattern that should/can be extracted?" I see this styling as temporary, in that I hope we'll have a reusable columnar-actions pattern out of `frontend-shared` in the near- to medium-term. (Such a pattern already exists in `frontend-shared`, but it's still in flux as to how we'd provide this as part of the public SASS/CSS API).

Anyway, if you feel strongly, we can use `.u-stretch`. I'm not that passionate!